ProjectHub2/services/facebookService.js

```diff
@@ -24,90 +19,127 @@ function makeApiCall(oauthOptions, callback, hashtag) {
     });
 }
 
-// ~~~~~~~~~~~~~~~~~~~~~~ posts ~~~~~~~~~~~~~~~~~~~~~~~
+// ~~~~~~~~~~~~~~~~~~~~~~ photos ~~~~~~~~~~~~~~~~~~~~~~~
 
-function getPagePosts(options, callback) {
+function getPhotoPosts(options) {
     var oauthOptions = {
         url: 'https://graph.facebook.com/v2.6/' + options['pageId'] + '?fields=albums{photos{created_time,id,name,likes.limit(1).summary(true),comments.limit(1).summary(true),images, from}}&access_token=' + options['appId'] + '|' + options['appSecret'],
         JSON: true
     };
-    makeApiCall(oauthOptions, callback);  
+    makeApiCall(oauthOptions, makePhotoArray);  
 }
 
-function printPagePosts(data) {
-    console.log('\n\n\n')
-    for (var i = 0; i < data['albums']['data'].length; i++) // albums
-    {
-       console.log('~~~~~~~~~~~~~~~~~~~~~~~~ album: ', i);
-       for (var j = 0; j < data['albums']['data'][i]['photos']['data'].length; j++) { // items in albums
-           console.log('-------------------');
-           console.log('id:', data['albums']['data'][i]['photos']['data'][j]['id']);
-           console.log('created_time:', data['albums']['data'][i]['photos']['data'][j]['created_time']);
-           console.log('from: ', data['albums']['data'][i]['photos']['data'][j]['from']);
-            if (data['albums']['data'][i]['photos']['data'][j].hasOwnProperty('name'))
-                console.log('description:', data['albums']['data'][i]['photos']['data'][j]['name']);
-            else
-                console.log('description: ');
-             
-            console.log('url:', data['albums']['data'][i]['photos']['data'][j]['images'][0]['source']);
-            console.log('likes:', data['albums']['data'][i]['photos']['data'][j]['likes']['summary']['total_count']);
-            console.log('comments:', data['albums']['data'][i]['photos']['data'][j]['comments']['summary']['total_count']);
-        }
-    }
+function makePhotoObject(data){
+    var object = {};
+    var dateOptions = { year: 'numeric', month: '2-digit', day: '2-digit', 
+                        hour: '2-digit', minute: '2-digit', second: '2-digit' };
+    object['_id'] = 'fb' + data.id;
+    date = new Date(data.created_time);
+    object.date = date.toLocaleString('ko-KR', dateOptions);
+    object.from = data.from;
+    if (data.hasOwnProperty('name'))
```

Comment:
it is on company's policy to use `{` and `}` after ALL conditionals (`if`, `for`, `while` etc)